source/vibe/utils/dictionarylist.d

```diff
@@ -59,6 +59,12 @@ struct DictionaryList(VALUE, bool case_sensitive = true, size_t NUM_STATIC_FIELD
 		foreach (k, ref v; this) ret ~= FieldTuple(k, v);
 		return ret;
 	}
+	/// ditto
+	const(FieldTuple)[] toRepresentation() const {
+		const(FieldTuple)[] ret;
+		foreach (k, ref v; this) ret ~= const(FieldTuple)(k, v);
+		return ret;
+	}
```

Comment:
I tried doing that, but the compiler complains:

    ../../source/vibe/utils/dictionarylist.d(59): Error: cannot implicitly convert expression (v) of type const(FilePart) to inout(FilePart)
    ../../source/vibe/inet/webform.d(142): Error: template instance vibe.utils.dictionarylist.DictionaryList!(FilePart, true, 1LU) error instantiating

Something is wrong with FilePart's const-correctness, I guess.